SPRINT-STRUCTURE.md

```diff
@@ -1,115 +1,73 @@
 # Structure of a Sprint
 
-## Goals
+The purpose of this document is to:
 
-This index structure is intended to accomplish the following goals
+- Organize the content in the playbook for quick reference and discoverability
+- Provide content in a logical structure which reflects the engineering process
+- Extensible hierarchy to allow teams to share deep subject matter expertise
 
-1. Organize for quick reference and discoverability.
-2. Provide content in a logical structure which reflects the engineering process.
-3. Extensible hierarchy to allow teams to share deep subject matter expertise.
+## A Wek in the Life of a CSE Project
 
-## A Week In The Life Of
+### Before starting the project
 
-This layout structures the playbook content to make it easy day to day to find relevant resources during an Agile sprint.
+- [ ] [Discuss and start writing the Team Agreements](team-agreements/readme.md). Update these documents with any decisions made throughout the project
+  - [Working Agreement](team-agreements/working-agreements/readme.md)
+  - [Definition of Ready](team-agreements/definition-of-ready/readme.md)
+  - [Definition of Done](team-agreements/definition-of-done/readme.md)
+  - [Estimation](sprint-planning/estimation/readme.md)
+- [ ] [Set up the repository/repositories](source-control/readme.md)
+  - Decide on repository structure/s
+  - Add [README.md](resources/templates/README.md), [LICENSE](resources/templates/LICENSE), [CONTRIBUTING.md](resources/templates/CONTRIBUTING.md), .gitignore, etc
+- [ ] [Decide on a versioning strategy](source-control/versioning/readme.md)
```

Comment:
Are we sure we need a versioning strategy before we can start the project? All the other points, make sense to me in this list, versioning strategy can be defined later IMHO.

Reply:
moved to day 4 - that will balance out the days as well.